wsmaster/che-core-api-system/src/main/java/org/eclipse/che/api/system/server/SystemManager.java

```diff
@@ -106,6 +106,7 @@ private void doStopServices() {
   @PreDestroy
   @VisibleForTesting
   void shutdown() throws InterruptedException {
+    LOG.info("Synchronous shutdown requested");
```

Comment:
it should be cleanup, no ?

Reply:
why ? is it a problem to have this log ? it's only called once on shutdown, but might be very useful to check that the post-destroy method is called on SIGTERM, which wasn't the case last week (for an unknown reason, this was fixed when updating to the last master branch).

Reply:
it's more the level of the logger

Reply:
well, the log level used in line [91](https://github.com/davidfestal/che/blob/f39184e40f737d1eb786aa246b82ee261c06c8af/wsmaster/che-core-api-system/src/main/java/org/eclipse/che/api/system/server/SystemManager.java#L91) is also `INFO`, and the type of event is clearly the same, no ?

Reply:
It's my view on log events (there are other views ;-) I still think these events are not at INFO level.
People that are interested in a specific part want always to push to INFO level but many times it's more DEBUG info. I'm not sure it's interesting for end-user to see that by default.

Reply:
Fixed [here](https://github.com/eclipse/che/pull/7243/commits/15642ff2b93258e06534cf71adb2de746e4809e2)